admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -150,15 +146,21 @@ main() {
         effect: NoSchedule'
     fi
 
-    TLS_PRIVATE_KEY=$(cat "$SERVER_KEY" | base64 -w 0) \
+    if [ -x "$(command -v envsubst)" ]; then
+        ENVSUBST="$(command -v envsubst) < $TEMPLATE_PATH"
+    else
+        ENVSUBST="docker run --rm -v $TEMPLATE_PATH:$TEMPLATE_PATH --env-file <(env) supinf/envsubst $TEMPLATE_PATH"
+    fi
+
+    env - TLS_PRIVATE_KEY=$(cat "$SERVER_KEY" | base64 -w 0) \
```

Comment:
Let's conserve `PATH="$PATH"` in the environment too. For example `envsubst` or `docker` might not be in a standard location.